Source/Tools/FEXLoader/FEXLoader.cpp

```diff
@@ -150,47 +156,40 @@ void InterpreterHandler(fextl::string* Filename, const fextl::string& RootFS, fe
   // Is the file large enough for shebang
   if (ReadSize <= 2) {
     close(FD);
-    return;
+    return true;
   }
 
   // Handle shebang files
   if (Data[0] == '#' && Data[1] == '!') {
-    fextl::string InterpreterLine {Data.begin() + 2, // strip off "#!" prefix
-                                   std::find(Data.begin(), Data.end(), '\n')};
-    fextl::vector<fextl::string> ShebangArguments {};
+    std::string_view InterpreterLine {Data.begin() + 2, // strip off "#!" prefix
+                                      std::find(Data.begin(), Data.end(), '\n')};
+    fextl::vector<std::string_view> ShebangArguments {};
 
     // Shebang line can have a single argument
-    fextl::istringstream InterpreterSS(InterpreterLine);
-    fextl::string Argument;
-    while (std::getline(InterpreterSS, Argument, ' ')) {
-      if (Argument.empty()) {
-        continue;
+    using namespace std::literals;
+    auto Begin = InterpreterLine.begin();
+    auto ArgEnd = Begin;
+    auto End = InterpreterLine.end();
+    const auto Delim = " "sv;
+    for (; ArgEnd != End && Begin != End; Begin = ArgEnd + 1) {
+      // Find the end
```

Comment:
This comment was somewhat helpful with the long `find_first_of` expression after, but once find is used this is actually more confusing than helpful. (Which end? End of the line or of the current word?)